integrations/chroma/src/haystack_integrations/document_stores/chroma/document_store.py

```diff
@@ -79,9 +83,21 @@ def __init__(
         self._persist_path = persist_path
         self._distance_function = distance_function
         # Create the client instance
-        if persist_path is None:
-            self._chroma_client = chromadb.Client()
+        # Ensure only one of persist_path or host is provided
+        if persist_path and host:
+            error_message = "You cannot provide both `persist_path` and `host`. Please choose one."
```

Comment:
```suggestion
        if persist_path and (host or port is not None):
            error_message = "You must specify `persist_path` for local persistent storage or, alternatively, `host` and `port` for remote HTTP client connection. You cannot specify both options."
```

let's check also `port`.
probably the `error_message` should be split over multiple lines.